R/gghighlight.R

```diff
@@ -126,10 +136,14 @@ gghighlight <- function(...,
 VERY_SECRET_COLUMN_NAME <- sym("highlight..........")
 
 #' @export
-ggplot_add.gg_highlighter <- function(object, plot, object_name) {
+ggplot_add.gg_highlighter <- function(object, plot, ...) {
```

Comment:
Oh, wow. TIL: the signature of a method doesn't need to match with the signature of the generic method.

`function(object, plot, object_name)` (until v3.5.2) causes this error, but using `...` doesn't

```
❯ checking S3 generic/method consistency ... WARNING
  ggplot_add:
    function(object, plot, ...)
  ggplot_add.gg_highlighter:
    function(object, plot, object_name)
  See section ‘Generic functions and methods’ in the ‘Writing R
  Extensions’ manual.
```